Lusas_Adapter/AdapterActions/Execute.cs

```diff
@@ -67,6 +68,9 @@ public bool RunCommand(NewModel command)
             m_LusasApplication.fileOpen("%PerMachineAppDataPlatform%\\config\\AfterNewModel");
             d_LusasData.setAnalysisCategory("3D");
 
+            string fileName = @"sections.csv";
+            File.Delete(fileName);
```

Comment:
Given this will delete the file from the directory the code is being executed from, is that definitely going to always be the location of this file across all use cases for deleting the file?

I'm also not sure on the change log PR title. This does remove the warning for overwriting the sections no doubt, but the method of doing so is just deleting the original sections.

Is there a workflow for users wanting to keep the sections previously made and appending them with a new model rather than overwriting? There may not be, I'm just asking questions cause I don't know! :smile:

Reply:
- The `sections.csv` is always saved in the location of the model file - unless the user relocates it and repoints the directory within Lusas
- I'll update the title and change log
- We have discussed expanding the execute function so there is a `NewModel` and clean wipe (it's a similar issue in Robot where the section library is replicated to the new model
- The main drive for this is so https://github.com/BHoM/Test_Toolkit/pull/323 works seamlessly without raising a warning between runs